src/EnergyPlus/BranchNodeConnections.cc

```diff
@@ -665,6 +665,7 @@ namespace BranchNodeConnections {
 					}
 				}
 				if ( ! IsValid ) {
+					if (NodeConnections(Loop1).ObjectType == "COILSYSTEM:INTEGRATEDHEATPUMP:AIRSOURCE") continue;
```

Comment:
Ugh.  Super-specialization for just this one component.  Looks uber-klugy.  Maybe it's the best path forward, as I see below the same sort of thing was done for the unglazed transpired solar collector, but it certainly doesn't _look_ great.  Continuing review...